.github/workflows/on-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: Changelog
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened, labeled, unlabeled]
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Check Changelog Update
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Changelog check
+        uses: Zomzog/changelog-checker@v1.0.0
+        with:
+          fileName: CHANGELOG.md # default `CHANGELOG.adoc`
```

Comment:
I think we have CHANGELOGs in `.rst` format still. This will enforce only markdown, yeah?

Reply:
In this repo here (onna/open-api) we use Markdown 😄  -> https://github.com/onna/open-api/blob/master/CHANGELOG.md.

😄 

If we want to use it for *rst* we can change it to *Changelog.rst* and it will check for *rst* based changelogs 😄